<ui version="4.0" >
 <class>KTouchUserSetupDialog</class>
 <widget class="QDialog" name="KTouchUserSetupDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>324</width>
    <height>179</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Setup Users</string>
  </property>
  <property name="modal" >
   <bool>true</bool>
  </property>
  <layout class="QHBoxLayout" >
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item>
    <widget class="QListWidget" name="userListWidget" />
   </item>
   <item>
    <layout class="QVBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <widget class="KPushButton" name="addBtn" >
       <property name="text" >
        <string>&amp;Add</string>
       </property>
       <property name="shortcut" >
        <string>Ins, Alt+A</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="KPushButton" name="removeBtn" >
       <property name="text" >
        <string>&amp;Remove</string>
       </property>
       <property name="shortcut" >
        <string>Del, Alt+R</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType" >
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="KPushButton" name="closeBtn" >
       <property name="text" >
        <string>&amp;Close</string>
       </property>
       <property name="shortcut" >
        <string>Alt+C</string>
       </property>
       <property name="default" >
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
 <customwidgets>
  <customwidget>
   <class>KPushButton</class>
   <extends>QPushButton</extends>
   <header>kpushbutton.h</header>
  </customwidget>
 </customwidgets>
 <includes>
  <include location="local" >kpushbutton.h</include>
 </includes>
 <resources/>
 <connections/>
</ui>
